grammars/css.cson

```diff
@@ -1913,7 +1913,8 @@
             'match': '''(?x)
               (-?(?!\\d)(?:[\\w-]|[^\\x00-\\x7F]|\\\\(?:[0-9a-fA-F]{1,6}|.))+)
               \\s*
-              (?=[~|^\\]$*=]|/\\*)
```

Comment:
Thanks for your prompt reply. Watching my daughter so I'll give it a try later. Thanks again.